lib/debloater/cli.rb

```diff
@@ -0,0 +1,92 @@
+require 'debloater/connection'
+require 'debloater/engine'
+require 'io/console'
+
+module Debloater
+  class CLI
+    DEFAULTS = {
+      connection: {
+        host:     'localhost',
+        port:     5432,
+        user:     'postgres',
+        password: nil,
+        dbname:   nil,
+      },
+      engine: {
+        confirm:    true,
+        min_mb:     50,
+        max_density: 0.75,
```

Comment:
These values aren't line up... if you're going to align stuff this way, may as well align them together ;-)